src/css/components/_directory.scss

```diff
@@ -0,0 +1,196 @@
+.directory {
```

Comment:
Any way we can update our margins, padding and rems to use the new mixins before this goes live?